app/models/audit_log.rb

```diff
@@ -1,5 +1,3 @@
 class AuditLog < ApplicationRecord
   belongs_to :user
-
-  validates :user_id, presence: true
```

Comment:
Do we view these anywhere in the app? I tried searching for views but couldn't find any. Will not having a user would break any assumptions we might have made?

Reply:
Good call, let me check and get back to you :)

Reply:
Yeah, I would have broken the the moderator's page. Fixed with https://github.com/thepracticaldev/dev.to/pull/7275/commits/d34e007fd77f91b988ff3f73bb0a13930096151e

Reply:
Thanks for tracking that down, I figured they were shown somewhere!